.github/workflows/build-and-publish-rule-book.yaml

```diff
@@ -74,7 +74,7 @@ jobs:
 
   check_pages:
     needs: [matrix-setup, digital_build, printable_build]
-    name: 🔍 Page count check ${{ matrix.display }}
+    name: ↔️ Page count check ${{ matrix.display }}
```

Comment:
What's with the "middleware" emoji?

Reply:
It compares left and right sides.

Reply:
Any other option? It looks terrible.

<img width="1189" height="502" alt="image" src="https://github.com/user-attachments/assets/cf82826c-fbd3-4141-bd59-0ef00e81a2cf" />

Reply:
I agree that it doesn't look pretty. Better fit for this than the Zoom icon though.

The non-emoji double-sided arrow doesn't fit the style of the rest of checks.

Maybe something based on ⭾?

Reply:
No arrows, please, let's make it look nice and pleasant. An option: 🧐